edx_proctoring/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import
 
-__version__ = '0.18.2'
+__version__ = '0.18.3'
```

Comment:
I'd argue that this is a big enough feature addition to justify a minor version increase.

I'd really like to argue that having fully-used-in-production versions that are < 1 is kind of silly, and we should just bite the bullet and declare a 1.0.0 release one of these days, but that may be overkill.